File: api/AutomationPortal/DB/Entity/Team.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace AutomationPortal.DB.Entity
{
    public class Team
    {
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        public bool IsDefault { get; set; }
        public string Email { get; set; }

        public int CustomerId { get; set; }
        public Customer Customer { get; set; }

        public virtual ICollection<TeamCustomFieldValue> CustomFieldValue { get; set; }
    }
}

File: automation-portal/src/app/team/team.ts
import { CustomFieldValue } from "../custom-field/custom-field-value";

export class Team {
    id: number = 0;
    name: string = '';
    isDefault: boolean = false;
    email: string = '';
    customerId: number = 0;
    customFieldValue: CustomFieldValue[] = new Array<CustomFieldValue>();
}
